src/main/webapp/script.js

```diff
@@ -18,3 +18,48 @@ function createElementByTag(text, tag) {
   element.innerText = text;
   return element;
 }
+
+/**
+ * parse.js
```

Comment:
`script.js` was created to write functions which are used for several `.js` files.
Maybe it will be better to add all this data in `parse.js`?